system/lib/libc/musl/src/stdio/__lockfile.c

```diff
@@ -10,6 +10,8 @@ int __lockfile(FILE *f)
 	while ((owner = a_cas(&f->lock, 0, tid)))
 		__wait(&f->lock, &f->waiters, owner, 1);
 	return 1;
+#else
```

Comment:
may as well share the `return 1;` for both code paths, which means we don't need the `else`.

Reply:
Done.